e2e/pod.go

```diff
@@ -210,3 +210,25 @@ func deletePodWithLabel(label, ns string, skipNotFound bool) error {
 	}
 	return err
 }
+
+// getMountPath fetches the path on an app where volume is mounted.
+func getMountPath(app *v1.Pod) string {
+	path := app.Spec.Containers[0].VolumeMounts[0].MountPath
+	return path
+}
+
+// calculateSha512sum returns the sha512sum of a file inside a pod.
+func calculateSha512sum(f *framework.Framework, app *v1.Pod, filePath string, opt *metav1.ListOptions) (string, error) {
```

Comment:
nit: `calculateSHA512sum()` might be better

Reply:
Yup, that will be better :+1: